docs/migration-from-hermione.md

```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
```

Comment:
you can also add information that for a seamless move in CI, we provide both `testplane` and `hermione` binaries

Reply:
I'd omit "So":

To migrate from Hermione to Testplane you'll need to:

Not critical to me.